pkg/operator/guardbudget/guard_budget_controller.go

```diff
@@ -111,11 +112,11 @@ func (c *GuardBudgetController) sync() error {
 func (c *GuardBudgetController) checkGuardBudget() error {
 	deploymentScaledDown := false
 	quorumGuardUnmanaged := true
-	etcdQGScale, mcoDeployErr := c.kubeClient.AppsV1().Deployments("openshift-machine-config-operator").GetScale("etcd-quorum-guard", metav1.GetOptions{})
+	etcdQG, mcoDeployErr := c.kubeClient.AppsV1().Deployments("openshift-machine-config-operator").Get("etcd-quorum-guard", metav1.GetOptions{})
```

Comment:
don't we want this to be deleted in the near future?  I can see it exists on both, but it will fail soon.